crates/bevy_pbr/src/decal/clustered.rs

```diff
@@ -97,6 +98,67 @@ pub struct ClusteredDecal {
     pub tag: u32,
 }
 
+/// Cubemap layout defines the order of images in a packed cubemap image.
+#[derive(Default, Reflect, Debug, Clone, Copy)]
+pub enum CubemapLayout {
```

Comment:
i added support for 4 standard cubemap layouts. unity supports these plus some others, which we could add in future. 

i didn't add cubemap array image support, as i wanted to reuse the existing decal texture array for simplicity.

it's maybe not ideal to have this in the shader, but preprocessing would be significantly harder.

Reply:
I'm not a fan of this TBH -- I'd rather just use cubemap arrays -- but it's OK for now, as long as we put the code behind an `#ifdef` in the shader to avoid regressing our VGPR usage.